### PR TITLE
Hover same address on hover

### PR DESCRIPTION
**Pull request by maciekzygmunt**
image
image

### CODE REVIEW

**Comment by lucadonnoh** [comment]
looks great! let's make it a bit subtler like in etherscan tho
image

File: packages/frontend/src/components/project/ContractEntry.tsx [hunk 1]
```diff
@@ -99,11 +99,12 @@ export function ContractEntry({
             ))}
             {contract.links.map((x, i) => (
               <Link
+                data-role="etherscan-link"
                 key={i}
                 className={cx(
                   verificationStatus.contracts[x.address] === false &&
                     !x.isAdmin
```
Comment (torztomasz) [inline packages/frontend/src/components/project/ContractEntry.tsx:106]: `!x.isAdmin && 'text-red-300'`

**Review by torztomasz — APPROVED** [review]

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
is this checking the full address or just the text displayed on the website (first few chars and last few chars)? because if that's the case, it could happen that two addresses looks similar but are different and they should not be highlighted together

**Comment by maciekzygmunt** [comment]
> is this checking the full address or just the text displayed on the website (first few chars and last few chars)? because if that's the case, it could happen that two addresses looks similar but are different and they should not be highlighted together

It is checking the full address

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**